enterprise/frontend/src/embedding-sdk/types/theme/index.ts

```diff
@@ -78,7 +80,7 @@ export type MetabaseColor = keyof MetabaseColors;
  * Every non-optional properties here must have a default value defined
  * in DEFAULT_METABASE_COMPONENT_THEME at [default-component-theme.ts]
  */
-export interface MetabaseComponentTheme {
+export type MetabaseComponentTheme = {
```

Comment:
Changed this to a `type` so we can use this in the `useThemeSpecificSelectors`

Reply:
I think `interface` also works there, but yes `type` works

Reply:
Yeah, that's what I was thinking as well, but it doesn't work with the `MetabaseComponentThemeKey` because TS doesn't think it's a record for whatever reason